### ISSUE TITLE
运行出错Assertion failure in -[MGSSyntaxController loadSyntaxDefinitions], /Users/libokun/Downloads/Fragaria-master/MGSSyntaxController.m:243

### ISSUE THREAD

**Issue by derekzhan** [tbl00c/ZZUIHelper#2]
错误日志如下，请大哥帮忙看看，谢谢。
`
2019-05-27 14:23:40.830952+0800 ZZUIHelper[47325:2203436] *** Assertion failure in -[MGSSyntaxController loadSyntaxDefinitions], /Users/libokun/Downloads/Fragaria-master/MGSSyntaxController.m:243
2019-05-27 14:23:40.834005+0800 ZZUIHelper[47325:2203436] [General] An uncaught exception was raised
2019-05-27 14:23:40.834039+0800 ZZUIHelper[47325:2203436] [General] framework syntax definitions not found
2019-05-27 14:23:40.834202+0800 ZZUIHelper[47325:2203436] [General] (
	0   CoreFoundation                      0x00007fff528c7cfd __exceptionPreprocess + 256
	1   libobjc.A.dylib                     0x00007fff7cf6da17 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff528e2a1a +[NSException raise:format:arguments:] + 98
	3   Foundation                          0x00007fff54b81c85 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 194
	4   MGSFragaria                         0x000000010046898a -[MGSSyntaxController loadSyntaxDefinitions] + 314
	5   MGSFragaria                         0x0000000100468091 -[MGSSyntaxController insertSyntaxDefinitions] + 65
	6   MGSFragaria                         0x000000010046798b -[MGSSyntaxController init] + 107
	7   MGSFragaria                         0x000000010046788a __37+[MGSSyntaxController sharedInstance]_block_invoke + 58
	8   libdispatch.dylib                   0x00000001004cff1b _dispatch_client_callout + 8
	9   libdispatch.dylib                   0x00000001004d1baa _dispatch_once_callout + 87
	10  MGSFragaria                         0x0000000100467835 +[MGSSyntaxController sharedInstance] + 117
	11  MGSFragaria                         0x000000010044a21d -[SMLSyntaxColouring syntaxDictionary] + 125
	12  MGSFragaria                         0x000000010044a17c -[SMLSyntaxColouring applySyntaxDefinition] + 44
	13  MGSFragaria                         0x0000000100449460 -[SMLSyntaxColouring initWithDocument:] + 2176
	14  MGSFragaria                         0x000000010046ca34 -[MGSFragaria embedInView:] + 3092
	15  ZZUIHelper                          0x000000010005676a -[ZZCodeHFileViewController viewDidLoad] + 154
	16  AppKit                              0x00007fff5001adaf -[NSViewController _sendViewDidLoad] + 94
	17  AppKit                              0x00007fff5001a0b5 -[NSViewController _loadViewIfRequired] + 385
	18  AppKit                              0x00007fff50019eea -[NSViewController view] + 30
	19  AppKit                              0x00007fff50151083 -[NSTabViewController _goodTabViewContentSize] + 254
	20  AppKit                              0x00007fff50150d58 -[NSTabViewController viewDidLoad] + 83
	21  ZZUIHelper                          0x0000000100013f53 -[ZZCodeAreaViewController viewDidLoad] + 51
	22  AppKit                              0x00007fff5001adaf -[NSViewController _sendViewDidLoad] + 94
	23  AppKit                              0x00007fff5001a0b5 -[NSViewController _loadViewIfRequired] + 385
	24  AppKit                              0x00007fff50019eea -[NSViewController view] + 30
	25  AppKit                              0x00007fff5002e326 -[_NSSplitViewItemViewWrapper wrapView] + 91
	26  AppKit                              0x00007fff5001b0dd -[NSSplitViewController _setupSplitView] + 590
	27  AppKit                              0x00007fff5001ae5d -[NSSplitViewController viewDidLoad] + 150
	28  AppKit                              0x00007fff5001adaf -[NSViewController _sendViewDidLoad] + 94
	29  AppKit                              0x00007fff5001a0b5 -[NSViewController _loadViewIfRequired] + 385
	30  AppKit                              0x00007fff50019eea -[NSViewController view] + 30
	31  AppKit                              0x00007fff5010a36f -[NSWindow _contentViewControllerChanged] + 109
	32  Foundation                          0x00007fff54acf941 -[NSObject(NSKeyValueCoding) setValue:forKey:] + 331
	33  AppKit                              0x00007fff50016e28 -[NSWindow setValue:forKey:] + 111
	34  AppKit                              0x00007fff4ff6aa43 -[NSIBUserDefinedRuntimeAttributesConnector establishConnection] + 637
	35  AppKit                              0x00007fff4fe8de13 -[NSIBObjectData nibInstantiateWithOwner:options:topLevelObjects:] + 1436
	36  AppKit                              0x00007fff5001f1c7 -[NSNib _instantiateNibWithExternalNameTable:options:] + 679
	37  AppKit                              0x00007fff5001ee23 -[NSNib _instantiateWithOwner:options:topLevelObjects:] + 136
	38  AppKit                              0x00007fff506ed61f -[NSStoryboard instantiateControllerWithIdentifier:] + 236
	39  AppKit                              0x00007fff4fe7ed31 NSApplicationMain + 702
	40  ZZUIHelper                          0x0000000100008872 main + 34
	41  libdyld.dylib                       0x00007fff7e73b3d5 start + 1
	42  ???                                 0x0000000000000003 0x0 + 3
)
2019-05-27 14:23:40.839870+0800 ZZUIHelper[47325:2203436] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'framework syntax definitions not found'
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff528c7cfd __exceptionPreprocess + 256
	1   libobjc.A.dylib                     0x00007fff7cf6da17 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff528e2a1a +[NSException raise:format:arguments:] + 98
	3   Foundation                          0x00007fff54b81c85 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 194
	4   MGSFragaria                         0x000000010046898a -[MGSSyntaxController loadSyntaxDefinitions] + 314
	5   MGSFragaria                         0x0000000100468091 -[MGSSyntaxController insertSyntaxDefinitions] + 65
	6   MGSFragaria                         0x000000010046798b -[MGSSyntaxController init] + 107
	7   MGSFragaria                         0x000000010046788a __37+[MGSSyntaxController sharedInstance]_block_invoke + 58
	8   libdispatch.dylib                   0x00000001004cff1b _dispatch_client_callout + 8
	9   libdispatch.dylib                   0x00000001004d1baa _dispatch_once_callout + 87
	10  MGSFragaria                         0x0000000100467835 +[MGSSyntaxController sharedInstance] + 117
	11  MGSFragaria                         0x000000010044a21d -[SMLSyntaxColouring syntaxDictionary] + 125
	12  MGSFragaria                         0x000000010044a17c -[SMLSyntaxColouring applySyntaxDefinition] + 44
	13  MGSFragaria                         0x0000000100449460 -[SMLSyntaxColouring initWithDocument:] + 2176
	14  MGSFragaria                         0x000000010046ca34 -[MGSFragaria embedInView:] + 3092
	15  ZZUIHelper                          0x000000010005676a -[ZZCodeHFileViewController viewDidLoad] + 154
	16  AppKit                              0x00007fff5001adaf -[NSViewController _sendViewDidLoad] + 94
	17  AppKit                              0x00007fff5001a0b5 -[NSViewController _loadViewIfRequired] + 385
	18  AppKit                              0x00007fff50019eea -[NSViewController view] + 30
	19  AppKit                              0x00007fff50151083 -[NSTabViewController _goodTabViewContentSize] + 254
	20  AppKit                              0x00007fff50150d58 -[NSTabViewController viewDidLoad] + 83
	21  ZZUIHelper                          0x0000000100013f53 -[ZZCodeAreaViewController viewDidLoad] + 51
	22  AppKit                              0x00007fff5001adaf -[NSViewController _sendViewDidLoad] + 94
	23  AppKit                              0x00007fff5001a0b5 -[NSViewController _loadViewIfRequired] + 385
	24  AppKit                              0x00007fff50019eea -[NSViewController view] + 30
	25  AppKit                              0x00007fff5002e326 -[_NSSplitViewItemViewWrapper wrapView] + 91
	26  AppKit                              0x00007fff5001b0dd -[NSSplitViewController _setupSplitView] + 590
	27  AppKit                              0x00007fff5001ae5d -[NSSplitViewController viewDidLoad] + 150
	28  AppKit                              0x00007fff5001adaf -[NSViewController _sendViewDidLoad] + 94
	29  AppKit                              0x00007fff5001a0b5 -[NSViewController _loadViewIfRequired] + 385
	30  AppKit                              0x00007fff50019eea -[NSViewController view] + 30
	31  AppKit                              0x00007fff5010a36f -[NSWindow _contentViewControllerChanged] + 109
	32  Foundation                          0x00007fff54acf941 -[NSObject(NSKeyValueCoding) setValue:forKey:] + 331
	33  AppKit                              0x00007fff50016e28 -[NSWindow setValue:forKey:] + 111
	34  AppKit                              0x00007fff4ff6aa43 -[NSIBUserDefinedRuntimeAttributesConnector establishConnection] + 637
	35  AppKit                              0x00007fff4fe8de13 -[NSIBObjectData nibInstantiateWithOwner:options:topLevelObjects:] + 1436
	36  AppKit                              0x00007fff5001f1c7 -[NSNib _instantiateNibWithExternalNameTable:options:] + 679
	37  AppKit                              0x00007fff5001ee23 -[NSNib _instantiateWithOwner:options:topLevelObjects:] + 136
	38  AppKit                              0x00007fff506ed61f -[NSStoryboard instantiateControllerWithIdentifier:] + 236
	39  AppKit                              0x00007fff4fe7ed31 NSApplicationMain + 702
	40  ZZUIHelper                          0x0000000100008872 main + 34
	41  libdyld.dylib                       0x00007fff7e73b3d5 start + 1
	42  ???                                 0x0000000000000003 0x0 + 3
)
libc++abi.dylib: terminating with uncaught exception of type NSException
(lldb) 
`
